# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fribidi/pkg-config-include-patch.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -u fribidi-0.10.4/fribidi.pc.in fribidi-0.10.4-fixed/fribidi.pc.in
--- fribidi-0.10.4/fribidi.pc.in	2002-01-03 16:31:24.000000000 +0100
+++ fribidi-0.10.4-fixed/fribidi.pc.in	2003-05-18 22:29:08.000000000 +0200
@@ -7,4 +7,4 @@
 Description: Unicode BiDirectional algorithm library
 Version: @VERSION@
 Libs: -L${libdir} -lfribidi
-Cflags: -I${includedir}/fribidi
+Cflags: -I${includedir} -I${includedir}/fribidi
Common subdirectories: fribidi-0.10.4/tests and fribidi-0.10.4-fixed/tests
Common subdirectories: fribidi-0.10.4/unidata and fribidi-0.10.4-fixed/unidata
